diff --git a/src/tools/lcc/src/c.h b/src/tools/lcc/src/c.h
index 43bec083..a7044d38 100644
--- a/src/tools/lcc/src/c.h
+++ b/src/tools/lcc/src/c.h
@@ -645,7 +645,7 @@ extern int process(char *);
 extern int findfunc(char *, char *);
 extern int findcount(char *, int, int);
 
-extern Tree constexpr(int);
+extern Tree constantexpr(int);
 extern int intexpr(int, int);
 extern Tree simplify(int, Type, Tree, Tree);
 extern int ispow2(unsigned long u);
diff --git a/src/tools/lcc/src/init.c b/src/tools/lcc/src/init.c
index 172d7c04..e63c5c59 100644
--- a/src/tools/lcc/src/init.c
+++ b/src/tools/lcc/src/init.c
@@ -192,7 +192,7 @@ static int initstruct(int len, Type ty, int lev) {
 	return n;
 }
 
-/* initializer - constexpr | { constexpr ( , constexpr )* [ , ] } */
+/* initializer - constantexpr | { constantexpr ( , constantexpr )* [ , ] } */
 Type initializer(Type ty, int lev) {
 	int n = 0;
 	Tree e;
diff --git a/src/tools/lcc/src/simp.c b/src/tools/lcc/src/simp.c
index ea26ab69..dfcb8483 100644
--- a/src/tools/lcc/src/simp.c
+++ b/src/tools/lcc/src/simp.c
@@ -182,7 +182,7 @@ static int subi(long x, long y, long min, long max, int needconst) {
 static int subd(double x, double y, double min, double max, int needconst) {
 	return addd(x, -y, min, max, needconst);
 }
-Tree constexpr(int tok) {
+Tree constantexpr(int tok) {
 	Tree p;
 
 	needconst++;
@@ -192,7 +192,7 @@ Tree constexpr(int tok) {
 }
 
 int intexpr(int tok, int n) {
-	Tree p = constexpr(tok);
+	Tree p = constantexpr(tok);
 
 	needconst++;
 	if (p->op == CNST+I || p->op == CNST+U)
diff --git a/src/tools/lcc/src/stmt.c b/src/tools/lcc/src/stmt.c
index 9c3bdbe9..356092bd 100644
--- a/src/tools/lcc/src/stmt.c
+++ b/src/tools/lcc/src/stmt.c
@@ -119,7 +119,7 @@ void statement(int loop, Swtch swp, int lev) {
 		       		static char stop[] = { IF, ID, 0 };
 		       		Tree p;
 		       		t = gettok();
-		       		p = constexpr(0);
+		       		p = constantexpr(0);
 		       		if (generic(p->op) == CNST && isint(p->type)) {
 		       			if (swp) {
 		       				needconst++;
